js/full-picture-render.js

```diff
@@ -0,0 +1,61 @@
+import {isEscapeKeydown} from './functions.js';
+
+const bigPicture = document.querySelector('.big-picture');
+const commentsTemplate = document.querySelector('#comments').content.querySelector('.social__comment');
+const socialComments = bigPicture.querySelector('.social__comments');
+const bigPictureCloseButton = bigPicture.querySelector('.big-picture__cancel');
+
+const onDocumentKeydown = (evt) => {
+  if (isEscapeKeydown(evt)) {
+    evt.preventDefault();
+    bigPicture.classList.add('hidden');
+    document.removeEventListener('keydown', onDocumentKeydown);
+    document.body.classList.remove('modal-open');
+  }
+};
+
+const renderBigPicture = (evt, photo) => {
+
+  evt.preventDefault();
+  document.body.classList.add('modal-open');
+
+  //#region hide
+  bigPicture.querySelector('.social__comment-count').classList.add('hidden');
+  bigPicture.querySelector('.comments-loader').classList.add('hidden');
+  //#endregion
+
+  //#region add data to template
+  bigPicture.querySelector('.big-picture__img').querySelector('img').src = photo['url'];
```

Comment:
```suggestion
  bigPicture.querySelector('.big-picture__img').querySelector('img').src = photo.url;
```

А почему не пользуешься точечной нотацией?) имхо так чуть короче и проще читается))

Reply:
При передаче ссылки на photo в функцию в другом модуле у меня не работала точечная нотация, поэтому чтобы код был единообразным, решил поменять в этом месте тоже на скобочную нотацию.